src/current/molt/migration-overview.md

```diff
@@ -0,0 +1,136 @@
+
+MOLT [Fetch](#fetch) and [Verify](#verify) are CLI-based to maximize control, automation, and visibility during the data load and replication stages.
+
+<table class="comparison-chart">
+  <tr>
+    <th>Tool</th>
+    <th>Usage</th>
+    <th>Tested and supported sources</th>
+    <th>Release status</th>
+  </tr>
+  <tr>
+    <td class="comparison-chart__feature"><a href="#schema-conversion-tool"><b>Schema Conversion Tool</b></a></td>
+    <td>Schema conversion</td>
+    <td>PostgreSQL, MySQL, Oracle, SQL Server</td>
+    <td>GA (Cloud only)</td>
+  </tr>
+  <tr>
+    <td class="comparison-chart__feature"><a href="#fetch"><b>Fetch</b></a></td>
+    <td>Initial data load; optional continuous replication</td>
+    <td>PostgreSQL 11-14, MySQL 5.7-8.0+, CockroachDB</td>
+    <td>GA</td>
+  </tr>
+  <tr>
+    <td class="comparison-chart__feature"><a href="#verify"><b>Verify</b></a></td>
+    <td>Schema and data validation</td>
+    <td>PostgreSQL 12-14, MySQL 5.7-8.0+, CockroachDB</td>
+    <td><a href="{% link {{ site.current_cloud_version }}/cockroachdb-feature-availability.md %}">Preview</a></td>
+  </tr>
+</table>
+
+### Schema Conversion Tool
+
+The [MOLT Schema Conversion Tool]({% link cockroachcloud/migrations-page.md %}) converts a source database schema to a CockroachDB-compatible schema. The tool:
+
+- Identifies [unimplemented features]({% link molt/migration-strategy.md %}#unimplemented-features-and-syntax-incompatibilities)
+- Rewrites unsupported [DDL syntax]({% link {{ site.current_cloud_version }}/sql-statements.md %}#data-definition-statements)
+- Applies CockroachDB [schema best practices]({% link molt/migration-strategy.md %}#schema-design-best-practices)
+
+### Fetch
+
+[MOLT Fetch]({% link molt/molt-fetch.md %}) performs the core data migration to CockroachDB. It supports:
+
+- [Multiple migration modes](#migration-modes) via `IMPORT INTO` or `COPY FROM`
+- Data movement via [cloud storage, local file servers, or direct copy]({% link molt/molt-fetch.md %}#data-path).
+- [Concurrent data export]({% link molt/molt-fetch.md %}#best-practices) from multiple source tables and shards
+- [Continuous replication]({% link molt/molt-fetch.md %}#replicate-changes), enabling you to minimize downtime before cutover
+- [Schema transformation rules]({% link molt/molt-fetch.md %}#transformations)
+- Safe [continuation]({% link molt/molt-fetch.md %}#fetch-continuation) to retry failed or interrupted tasks from specific checkpoints
+- [Failback]({% link molt/molt-fetch.md %}#fail-back-to-source-database) to replicate changes from CockroachDB back to the original source via a secure changefeed
+
+### Verify
+
+[MOLT Verify]({% link molt/molt-verify.md %}) checks for data and schema discrepancies between the source database and CockroachDB. It performs:
+
+- Table structure verification
+- Column definition verification
+- Row-level data verification
+- Continuous, live, or one-time verification
```

Comment:
Though we support this, want to say this is not an oft-tested path and we've had issues with customers workign with this mode. CC @Jeremyyang920 and @rohan-joshi

Reply:
Commenting this line out for now.

Reply:
I think in this case we should remove the continuous and live portion until we actually GA and test everything